plugins/3MFReader/WorkspaceRow.qml

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2022 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+import QtQuick.Window 2.2
+
+import UM 1.5 as UM
+import Cura 1.1 as Cura
+
+Row
+{
+    property alias leftLabelText: leftLabel.text
+    property alias rightLabelText: rightLabel.text
+
+    width: parent.width
+    height: visible ? childrenRect.height : 0
+
+    UM.Label
+    {
+        id: leftLabel
+        text: catalog.i18nc("@action:label", "Type")
+        width: (parent.width / 4) | 0
```

Comment:
I don't really like the `| 0` pattern. It flooring the value, and I believe we use it so we don't have non-integer widths. I rather use `Math.round` because I think rounding makes more sense then flooring, and I think it is more readable. `| 0` is supposed to be a bit faster though, but I don't think it is going to be noticeable.
```suggestion
        width: Math.round(parent.width / 4)
```